doc/source/cluster/configure-manage-dashboard.md

```diff
@@ -0,0 +1,304 @@
+
+Configure these settings using the `RAY_GRAFANA_HOST`, `RAY_PROMETHEUS_HOST`, and `RAY_GRAFANA_IFRAME_HOST` environment variables when you start the Ray clusters.
+
+* Set `RAY_GRAFANA_HOST` to an address that the head node can use to access Grafana. Head node does health checks on Grafana on the backend.
+* Set `RAY_PROMETHEUS_HOST` to an address the head node can use to access Prometheus.
+* Set`RAY_GRAFANA_IFRAME_HOST` to an address that the user's browsers can use to access Grafana and embed visualizations. If not set, `RAY_GRAFANA_IFRAME_HOST` uses the value of `RAY_GRAFANA_HOST`.
+
+For example, if the IP of the head node is 55.66.77.88 and Grafana is hosted on port 3000. Set the value to `RAY_GRAFANA_HOST=55.66.77.88:3000`.
+
+If all the env vars are set properly, you should be able to see time-series metrics in {ref}`Ray Dashboard <observability-getting-started>`.
+
+#### Alternate Prometheus host location
+By default, we assume Prometheus is hosted at `localhost:9090`. You can choose to run Prometheus on a non-default port or on a different machine. When doing so, you should make sure that Prometheus can scrape the metrics from your ray nodes following instructions {ref}`here <scrape-metrics>`.
+
+Then, configure `RAY_PROMETHEUS_HOST` env var properly as stated above. For example, if Prometheus is hosted at port 9000 on a node with ip 55.66.77.88, One should set the value to `RAY_PROMETHEUS_HOST=http://55.66.77.88:9000`.
+
+
+#### Alternate Grafana host location
+By default, we assume Grafana is hosted at `localhost:3000` You can choose to run Grafana on a non-default port or on a different machine as long as the head node and the browsers of dashboard users can access it.
+
+If Grafana is exposed via nginx ingress on a Kubernetes cluster, the following line should be present in the Grafana ingress annotation:
+
+```yaml
+  nginx.ingress.kubernetes.io/configuration-snippet: |
+      add_header X-Frame-Options SAMEORIGIN always;
+```
+
+When both Grafana and Ray cluster are on the same Kubernetes cluster, it is important to set `RAY_GRAFANA_HOST` to the external URL of the Grafana ingress.
+
+
+
+#### User authentication for Grafana
+When the Grafana instance requires user authentication, the following settings have to be in its `configuration file <https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/>`_ to correctly embed in Ray dashboard:
+
+```ini
+  [security]
+  allow_embedding = true
+  cookie_secure = true
+  cookie_samesite = none
+```
+
+#### Troubleshooting
+
+##### Getting Grafana to use the Ray configurations when installed via homebrew on macOS X
+
+With homebrew, Grafana is installed as a service that is automatically launched for you.
+Therefore, to configure these services, you cannot simply pass in the config files as command line arguments.
+
+Instead, update `/usr/local/etc/grafana/grafana.ini` file so that it matches the contents of `/tmp/ray/session_latest/metrics/grafana/grafana.ini`.
```

Comment:
```suggestion
Instead, update `/usr/local/etc/grafana/grafana.ini` file so that it matches the contents of `/tmp/ray/session_latest/metrics/grafana/grafana.ini`.
```
```suggestion
Instead, update the `/usr/local/etc/grafana/grafana.ini` file so that it matches the contents of `/tmp/ray/session_latest/metrics/grafana/grafana.ini`.
```